src/components/one-time-donation/components/PriceRadioGroup.tsx

```diff
@@ -0,0 +1,45 @@
+import { useField } from 'formik'
+import { useTranslation } from 'next-i18next'
+import { FormControl, FormHelperText, RadioGroup, Typography } from '@mui/material'
+import { TranslatableField, translateError } from 'common/form/validation'
+import PriceRadioButton from './PriceRadioButton'
+import Stripe from 'stripe'
+import { money } from 'common/util/money'
+
+export type RadioGroupProps = {
+  name: string
+  options: Stripe.Price[] | undefined
```

Comment:
Maybe the type here is different

```tsx
options: { values: 'card' | 'bank', label: string }[]
```

Somewhere we have a mismatch

Reply:
This is happening, because the same radio group is used for the Third Step later. I will see how to make the whole group more reusable so we can use it later on too.